### ISSUE TITLE
Stray console.log in latest release [v9.1.3]

### ISSUE THREAD

**Issue by matt-morris** [imgix/react-imgix#795]
**Describe the bug**
Release 9.1.3 has a console.log in the build. 

**To Reproduce**
Install v9.1.3 and use one of the components.

https://codesandbox.io/s/sleepy-violet-fygb6?file=/src/App.js

The relevant line is in `react-imgix/lib/HOCs/shouldComponentUpdateHOC.js`

```js
_defineProperty(_assertThisInitialized(_this), "shouldComponentUpdate", function (nextProps) {
    var props = _this.props;
    console.log(props, nextProps);
    (0, _common.warning)(nextProps.onMounted == _this.props.onMounted, "props.onMounted() is changing between renders. This is probably not intended. Ensure that a class method is being passed to Imgix rather than a function that is created every render. If this is intended, ignore this warning.");
```

I don't actually see it in your code or this would be a PR instead, so maybe it's an artifact of your build process? Anyway, let me know if I can do anything else to help. 

Steps to reproduce the behaviour:

1.  Go to https://codesandbox.io/s/sleepy-violet-fygb6?file=/src/App.js
2.  Open the console
3.  See error

**Expected behaviour**
No extraneous console messages

**Information:**

- react-imgix version: 9.1.3

**Comment by sherwinski** [comment]
Hey @matt-morris, thanks for pointing this out to us! We're in the process of releasing a fix now 👍

**Comment by sherwinski** [comment]
Looks to be resolved in [v9.1.4](https://github.com/imgix/react-imgix/releases/tag/v9.1.4) (confirmed against your codesandbox) so I'm going to close this issue now. Thanks again!

**Comment by matt-morris** [comment]
Cheers. Thanks for a quick response 👍
